sdk/cargo-build-bpf/src/main.rs

```diff
@@ -44,6 +45,7 @@ impl Default for Config<'_> {
                 .join("sdk")
                 .join("bpf"),
             bpf_out_dir: None,
+            bpf_tools_version: "1.21",
```

Comment:
Perhaps something like this would be a better default?
```suggestion
            bpf_tools_version: "(unknown)",
```